test/index.test.js

```diff
@@ -107,7 +107,80 @@ test('arg truthy correctly parsed', function(t) {
   t.end();
 });
 
-test('env var substition throws on missing env vars', function(t) {
+test('arg same keys', function(t) {
+  resetArgv();
+  process.argv.push('--afoo=new-value');
+
+  let config = loadConfig(__dirname + '/fixture/one');
+  t.equal(config.afoo, 'new-value', 'last arg wins');
+
+  t.end();
+});
+
+test('args unknown', function(t) {
+  resetArgv();
+
+  let config = loadConfig(__dirname + '/fixture/one');
+  t.equal(config.asome, undefined, 'not existing arg is undefined');
+
+  t.end();
+});
+
+test('args whitespaces in assignment', function(t) {
```

Comment:
this does not test "whitespaces in assignment" because you've pushed the whole thing into a single argv slot

Reply:
TIL: this is actually somewhat valid if `=` is involved. E.g.

```
$ node index.js --foo="    spaaaace"
```

Results in argv containing `'--foo=    spaaaace'`

Reply:
Aha!